rp_redcap/tests/test_tasks.py

```diff
@@ -844,14 +880,18 @@ def test_save_patient_records_existing_update(self):
         patient_record = PatientRecord.objects.all()[0]
         patient_value = PatientValue.objects.all()[0]
 
-        self.assertEqual(patient_record.status, PatientRecord.INCOMPLETE_STATUS)
+        self.assertEqual(
+            patient_record.post_operation_status,
+            PatientRecord.INCOMPLETE_STATUS,
+        )
         self.assertEqual(patient_value.value, "new_value")
 
     def test_save_patient_records_new(self):
         date = utils.get_today()
         data = {
             "record_id": "1",
-            "asos2_crf_complete": PatientRecord.COMPLETE_STATUS,
```

Comment:
Just want to check here - is `asos2_crf_complete` no longer something of use? Is it being replaced entirely by `pre_operation_status` and `post_operation_status`?

Reply:
No, it's still in the payload but not really used, completeness is determined by the two sets of fields.